src/main/kotlin/Graph.kt

```diff
@@ -0,0 +1,48 @@
+import Graph.Companion.STOCKHOLM
+import Graph.Companion.UPPSALA
+
+class Graph {
+    companion object {
+        const val STOCKHOLM = "ID_STOCKHOLM"
+        const val UPPSALA = "ID_UPPSALA"
+    }
+
+    val user: User? by lazy {
+        User()
+    }
+
+    fun findShortestPath(node1: Node, node2: Node): Int {
+        return 0
+    }
+
+    fun getNodeById(id: String): Node {
+
+        // TODO: ?
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tokgozmusa/detekt-danger/blob/367f9cc3876b077123ef2d7b894f003e5f241fcc/src/main/kotlin/Graph.kt#L20"></span>This comment contains text that has been defined as forbidden in detekt.</td>
    </tr>
  </tbody>
</table>